### ISSUE TITLE
[Feature] Integration with beberlei/assert library

### ISSUE THREAD

**Issue by lisachenko** [lisachenko/php-deal#3]
Integration with @beberlei's assert library can give a nice tool for writing DbC conditions and assertions, see https://github.com/beberlei/assert#list-of-assertions. For example, 

```php
    /**
     * Deposits fixed amount of money to the account
     *
     * @param float $amount
     *
     * @Contract\Ensure("Assert\Assertion::integer($this->balance)")
     */
    public function deposit($amount)
    {
        $this->balance += $amount;
    }
```

**Comment by pdaw** [comment]
@lisachenko 

Good idea, I will create pull request with integration soon.

**Comment by lisachenko** [comment]
@pdaw Thanks for contribution! Released tag 0.3.0 with minor changes in your code.
